src/w3d/math/matrix4.h

```diff
@@ -476,3 +476,13 @@ __forceinline Vector4 operator*(const Matrix4 &a, const Vector4 &v)
         a[2][0] * v[0] + a[2][1] * v[1] + a[2][2] * v[2] + a[2][3] * v[3],
         a[3][0] * v[0] + a[3][1] * v[1] + a[3][2] * v[2] + a[3][3] * v[3]);
 }
+
+__forceinline int operator==(const Matrix4 &a, const Matrix4 &b)
```

Comment:
I know this is following the convention of the above but `a` and `b` are rubbish names `lhs` and `rhs` would be better.